action.yml

```diff
@@ -47,6 +47,9 @@ inputs:
   repo:
     description: "The repo to run against. Will use the the current repo if not specified."
     required: false
+  gh_timeout:
+    description: 'The amount of timeout for waiting for a runner to be online by GitHub.'
```

Comment:
as with the other point on similar text; maybe some more clarification

Reply:
Fixed here: [abf7339](https://github.com/omsf-eco-infra/gha-runner/pull/22/commits/abf7339bb5ab5273463e926ae4f0df2da053c064)